lib/rpush/daemon/store/redis.rb

```diff
@@ -120,6 +120,56 @@ def translate_integer_notification_id(id)
 
         private
 
+        ZRANGEBYSCORETHENREM_SCRIPT = <<-EOS
+          -- Redis lua script
+          local queue_key = KEYS[1]
+          local min = ARGV[1]
+          local max = ARGV[2]
+          redis.call('zrangebyscore', queue_key, min, max)
+          local return_results = redis.call('zremrangebyscore', queue_key, min, max)
```

Comment:
this doesnt make any sense to me. `zrangebyscore` is a getter. But we are not using the returned values anywhere. Whats the point of calling `zrangebyscore` ?

Reply:
The score is the epoch time the notification is created.